core/src/main/java/org/infinispan/xsite/BackupSenderImpl.java

```diff
@@ -243,50 +214,39 @@ public TakeSiteOfflineResponse takeSiteOffline(String siteName) {
       }
    }
 
+   private void updateOfflineSites(String siteName, long sendTimeMillis, Throwable throwable) {
+      OfflineStatus status = offlineStatus.get(siteName);
+      if (status != null && status.isEnabled()) {
+         if (isCommunicationError(throwable)) {
+            status.updateOnCommunicationFailure(sendTimeMillis);
+         } else if (!status.isOffline()) {
+            status.reset();
+         }
+      }
+   }
+
    private BackupResponse backupCommand(VisitableCommand command, List<XSiteBackup> xSiteBackups) throws Exception {
       XSiteReplicateCommand xsiteCommand = commandsFactory.buildSingleXSiteRpcCommand(command);
       //RpcManager is null for local caches.
       //TODO does it make sense to have xsite replication in a local cache?
       //TODO probably is better to create a LocalRpcManager implementation with xsite support only...
       return rpcManager == null ?
-             transport.backupRemotely(xSiteBackups, xsiteCommand):
+             transport.backupRemotely(xSiteBackups, xsiteCommand) :
```

Comment:
IMO no, it doesn't make sense to have xsite replication for a local cache, because each node sees different entries in the cache, and they get all mixed up during backup.

Reply:
created https://issues.jboss.org/browse/ISPN-10673
:+1: